events/desci-and-big-data-mgmt.toml

```
@@ -0,0 +1,125 @@
+# the name of your track or event
+name = "Decentralized Science (desci) and Big Data Management"
+
+# the github handle of the directly responsible individual for this event
+# (this person will coordinate with devent organizers)
+dri = "wesfloyd"
```

```suggestion
dri = "Wes Floyd, Niklas Rindtorff, Eleanor D."
```

Switched to full names (contrary to template) so it's more readable to the public. Added Eleanor D. per Wes's latest email.